joomla/cassiopeia
# Remove the white border on active pagination elements

Pull Request for Issue #107.

### Summary of Changes

This PR removes the white border on active pagination elements.
Removing grey border on disabled elements has consequences for disabled page-links as well. 

### Testing Instructions

- Joomla 4 with Testing Sample Data
- Go To /index.php/article-category-list
- Apply patch
- `npm run build:css`
- Notice borders around all pagination items


### Expected result

<img width="344" alt="Schermafbeelding 2020-09-24 om 22 20 31" src="https://user-images.githubusercontent.com/639822/94196467-b015ab80-feb4-11ea-81b3-61241fb4f604.png">


### Actual result



### Documentation Changes Required
